Frontend/ui-driver/src/Screens/AddVehicleDetailsScreen/ComponentConfig.purs

```diff
@@ -57,7 +57,7 @@ primaryButtonConfig state = let
       , width = MATCH_PARENT
       , height = (V 50)
       , background = Color.black900 
-      , margin = Margin 15 0 15 30
+      , margin = if (state.props.openHowToUploadManual&& (not state.data.cityConfig.uploadRCandDL)) then Margin 15 0 15 10 else Margin 15 0 15 30
```

Comment:
use above `imageUploadCondition` variable 
`imageUploadCondition = state.props.openHowToUploadManual && not state.data.cityConfig.uploadRCandDL`

And remove `--getValueFromConfig "imageUploadOptional" || state.props.isValidState` commented code
It was added for this condition only